tests/Functional/Schema/ExtensionTest.php

```diff
@@ -571,6 +580,193 @@ public function testDoesNotAllowExtendingANonObjectScalarType()
         '));
     }
 
+    // can add additional root operation types
```

Comment:
I'd drop these comments

Reply:
I'll keep this one, it's from the reference implementation.